### ISSUE TITLE
dns.env missing

### ISSUE THREAD

**Issue by andyjhall** [morph1904/TygerCaddy#96]
Looks like the dns.env file isnt included in the repo, this causes the below error.

**service caddy status:**

root@proxy:/apps/TygerCaddy/TygerCaddy# service caddy status
● caddy.service - Caddy HTTP/2 web server
Loaded: loaded (/etc/systemd/system/caddy.service; enabled; vendor preset: en
Active: failed (Result: resources) since Thu 2018-05-24 11:43:42 UTC; 4min 54
Docs: https://caddyserver.com/docs

May 24 11:43:42 proxy systemd[1]: caddy.service: Failed to load environment file
May 24 11:43:42 proxy systemd[1]: caddy.service: Failed to run 'start' task: No
May 24 11:43:42 proxy systemd[1]: caddy.service: Failed with result 'resources'.
May 24 11:43:42 proxy systemd[1]: Failed to start Caddy HTTP/2 web server.
May 24 11:43:42 proxy systemd[1]: caddy.service: Service hold-off time over, sch
May 24 11:43:42 proxy systemd[1]: caddy.service: Scheduled restart job, restart
May 24 11:43:42 proxy systemd[1]: Stopped Caddy HTTP/2 web server.
May 24 11:43:42 proxy systemd[1]: caddy.service: Start request repeated too quic
May 24 11:43:42 proxy systemd[1]: caddy.service: Failed with result 'resources'.
May 24 11:43:42 proxy systemd[1]: Failed to start Caddy HTTP/2 web server.
lines 1-15/15 (END)...skipping...
● caddy.service - Caddy HTTP/2 web server
Loaded: loaded (/etc/systemd/system/caddy.service; enabled; vendor preset: enabled)
Active: failed (Result: resources) since Thu 2018-05-24 11:43:42 UTC; 4min 54s ago
Docs: https://caddyserver.com/docs

May 24 11:43:42 proxy systemd[1]: caddy.service: Failed to load environment files: No such file or directory
May 24 11:43:42 proxy systemd[1]: caddy.service: Failed to run 'start' task: No such file or directory
May 24 11:43:42 proxy systemd[1]: caddy.service: Failed with result 'resources'.
May 24 11:43:42 proxy systemd[1]: Failed to start Caddy HTTP/2 web server.
May 24 11:43:42 proxy systemd[1]: caddy.service: Service hold-off time over, scheduling restart.
May 24 11:43:42 proxy systemd[1]: caddy.service: Scheduled restart job, restart counter is at 5.
May 24 11:43:42 proxy systemd[1]: Stopped Caddy HTTP/2 web server.
May 24 11:43:42 proxy systemd[1]: caddy.service: Start request repeated too quickly.
May 24 11:43:42 proxy systemd[1]: caddy.service: Failed with result 'resources'.
May 24 11:43:42 proxy systemd[1]: Failed to start Caddy HTTP/2 web server.

**Comment by sparky8251** [comment]
I'm testing for the 0.2 release today. I'm curious if this is with a fresh install of 0.2 or not.

We are trying to cut DNS verification out of 0.2 and push it to 0.3 and as of this morning everal changes were made to this end.

**Comment by andyjhall** [comment]
Yes completely fresh install, down the the VM being wiped out and rebuilt.

**Comment by sparky8251** [comment]
Ok. My testing this morning revolved around Docker.

Just edited the config file in the alpha 0.2 branch. Want to see if thats better from a fresh install now?
